sys/php/www.conf

```diff
@@ -106,23 +111,27 @@ pm = dynamic
 ; forget to tweak pm.* to fit your needs.
 ; Note: Used when pm is set to 'static', 'dynamic' or 'ondemand'
 ; Note: This value is mandatory.
+;pm.max_children = 5
 pm.max_children = 100
 
 ; The number of child processes created on startup.
 ; Note: Used only when pm is set to 'dynamic'
-; Default Value: min_spare_servers + (max_spare_servers - min_spare_servers) / 2
+; Default Value: (min_spare_servers + max_spare_servers) / 2
+;pm.start_servers = 2
```

Comment:
As the container is running on well defined env, having fixed resources, not shared with other processes I suggest to switch to a static forking model. Then start the max number of processes that can fill the RAM

Reply:
That need to be calculated accurately based on the resources used by a single request, I've seen lots of fluctuations on CPU and RAM. Need to spend some time on it to figure out the right value.

Reply:
A first, very rough, estimate could be made dividing the "RAM - a bit" by the PHP memory_limit setting